docker-compose.yml

```diff
@@ -11,25 +11,16 @@ services:
       - postgresql-data:/var/lib/postgresql/data
 
   reverse-proxy:
-    image: nginx:1.19.10-alpine
+    image: docker.pkg.github.com/thenewboston-developers/thenewboston-node/thenewboston-node-reverse-proxy:latest
```

Comment:
We need changes to `.github/workflows/master.yml` to make this image appear in that registry. Let's make them first as a separate PR. Once it works we can merge the rest.

You may push to master if needed to configure `.github/workflows/master.yml` for `thenewboston-node-reverse-proxy:latest`